src/scikit_hep_repo_review/processor.py

```diff
@@ -116,7 +116,6 @@ def process(package: Traversable) -> dict[str, list[Result]]:
 
 def as_simple_dict(results_dict: dict[str, list[Result]]) -> dict[str, dict[str, Any]]:
```

Comment:
```suggestion
def as_simple_dict(results_dict: dict[str, list[Result]]) -> dict[str, list[Any]]:
```

Should probably put a typed dict here.